### PR TITLE
chore(updated and organised repo)

### PR DESCRIPTION
**Pull request by Pavan-Nambi**

### CODE REVIEW

File: src/AWS_Account_Creation/Steps_for_AWS_Account_Creation.md [hunk 1]
```diff
@@ -44,21 +44,21 @@ Sign in Process
 5. Go to Accounts and Edit Alternate Contacts and click on Update
    <!-- !image -->
```
Comment (pooja-1405) [inline src/AWS_Account_Creation/Steps_for_AWS_Account_Creation.md:45]: Add the correct image.

Comment (Pavan-Nambi) [reply src/AWS_Account_Creation/Steps_for_AWS_Account_Creation.md:45]: u do urself

Comment (Pavan-Nambi) [inline src/AWS_Account_Creation/Steps_for_AWS_Account_Creation.md:45]: remove image tag

Comment (pooja-1405) [reply src/AWS_Account_Creation/Steps_for_AWS_Account_Creation.md:45]: okay

**Pull request merged**